packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -62,6 +62,14 @@ const SquadEmptyScreen = dynamic(
     ),
 );
 
+const SquadChecklistCard = dynamic(async () => {
```

Comment:
Just for my understanding why did you choose dynamic import when it's always rendered?
Seems to have no specific benefit that way.

Reply:
Alright cool makes sense though was just wondering the thoughts.
I'm not against it especially if it has benefits 🙏